### ISSUE TITLE
R: Create LDGMA Script for Open Data

### ISSUE THREAD

**Issue by croswell81** [NYCPlanning/edm-overview#678]
Create a script to be run monthly that will update Lower Density Growth Management Area (LDGMA) dataset. This is maintained by the GIS Team but has not been routinely updated.

LDGMA only applies to two geographies; **Borough of Staten Island** and **Bronx Community District 10** (water included)

 - [x] Clip the most up to date **zoning district** file to Staten Island and BX CD 10
 - [x] Clip the most up to date **commercial overlay** file to Staten Island and BX CD 10
    - use new monthly version from Bytes Production folder for above files.

 - [x] Filter clipped zoning district file for the following districts
    - [x] **Staten Island**: R1, R2, R3, R4*, R5, C3A and C1, C2, C4 
    - [x] **BX CD 10**: R1, R2, R3, R4*, R5, C3A and R6, R7, C1, C2 
    - (*R4A, R4-1 are specified in text but are covered in private roads text that includes R4 & R5)
 - [x] Dissolve all remaining zoning districts and merge the clipped commerical overlay districts into two polygons; one for SI and one for BX
 - [x] Merge the two polygons into one final output with two records, containing one 'Borough' field 

Update metadata to reflect the following:
 - [x] R6, R7, C1 and C2 districts in BX CD 10 are for applying the parking provisions of Article II, Chapter 5, and Article III, Chapter 6 only
 - [x] any zoning lot containing buildings accessed by private roads in R1, R2, R3, R4, R5 or C3A Districts (really only applies to other R4 and all R5 districts)

Data Management:
 - [x] Add a folder M:\GIS\BytesProduction
 - [x] Transfer updated file to SDE PROD
 - [x] Make sure lyr files are synced and update metadata
 - [x] Distribute to Open Data, Bytes, The Guide, AGOL, CARTO, etc.

**Comment by aferrar** [comment]
Submitted script results to Matt for review.

**Comment by croswell81** [comment]
Confirmed with Frank Ruchala (Zoning Division) that LDGMA should remain one single area and not be broken into subareas.

I proposed possibly having three LDGMA areas to reflect:

- LDGMA General area
- LDGMA, Private Roads: any zoning lot containing buildings accessed by private roads in R1, R2, R3, R4, R5 or C3A Districts 
- LDGMA, Parking: R6, R7, C1, C2 in BX CD 10 are for applying the parking provisions of Article II, Chapter 5, and Article III, Chapter 6 only

Also, confirmed Commercial Overlays should be included. Issue updated to reflect confirmations.
